src/bcs/deserializer.ts

```diff
@@ -82,6 +82,7 @@ export class Deserializer {
   }
 
   /**
+   * @deprecated use `deserializeOption` instead.
```

Comment:
I don't think this should be marked as deprecated,
`deserializeOption` returns `string | undefined` so it's not equivalent